themes/src/main/resources/theme/keycloak/welcome/index.ftl

```diff
@@ -28,13 +28,13 @@
           <main class="pf-v5-c-login__main">
             <header class="pf-v5-c-login__main-header">
               <#if localUser>
-                <h1 class="pf-v5-c-title pf-m-3xl">Create an administrative user</h1>
+                <h1 class="pf-v5-c-title pf-m-2xl">Create a temporary administrative user</h1>
                 <#if !successMessage?has_content>
-                  <p class="pf-v5-c-login__main-header-desc">To get started with ${productName}, you first create an administrative user.</p>
+                  <p class="pf-v5-c-login__main-header-desc">To get started with ${productName}, you first create a temporary administrative user. To harden security, create a permanent one and delete the temporary one.</p>
```

Comment:
I think this makes it a little more clear that the permanent user should be created later:
```suggestion
                  <p class="pf-v5-c-login__main-header-desc">To get started with ${productName}, you first create a temporary administrative user.  Later, to harden security, create a new permanent administrative user and delete the temporary user that was created during this setup.</p>
```

Reply:
@ssilvert LGTM, thanks!